### ISSUE TITLE
Between option is not working in search tab

### ISSUE THREAD

**Issue by Jay04653** [phpmyadmin/phpmyadmin#12501]
### Steps to reproduce
1. Open search form
2. Select between option
### Server configuration

**OS:** Linux Debian 7

**Web server:** Apache 2.4

**DB:** MySQL 5.6

**PHP:** 7.0.9

**phpMyAdmin:** 4.6.4
### Client configuration

**Web Browser:** Google Chrome

**Operating system:** Linux Ubuntu 16.04

**Comment by Jay04653** [comment]
![between](https://cloud.githubusercontent.com/assets/2777846/18007119/601c84be-6bab-11e6-957a-103d9d9e4718.png)

No errors in console.
Received JSON data on select: `{"redirect_flag":"1","success":false,"error":"<a class=\"hide\" id=\"update_recent_tables\" href=\"index.php?ajax_request=1&amp;recent_table=1&amp;token=b519d277877afa57e5bda4bc5b7ce83f\"><\/a>"}`

**Comment by Jay04653** [comment]
Also token always failed after selection.

**Comment by devenbansod** [comment]
Sorry, but I could not reproduce this.
Can you clear the cookies and try again (also try in Incognito/Private window once) ? Or try to reproduce this on our [demo server] (demo.phpmyadmin.net) (which I think is not working currently, but try again after sometime if you can) ?

**Comment by devenbansod** [comment]
Closed by mistake. Reopened.

**Comment by Jay04653** [comment]
Tried in incognito window, got the same error.
Actually I'am receiving this error on only in last version but maybe since 4.4 or even earlier.

Can't check it MySQL Server.
Don't know how cat I debug it. I do not have js errors it just not working.

Here code of broken popup message:
`<span class="ajax_notification dismissable" id="ajax_message_num_2" style="cursor: pointer; opacity: 0.955702;"><a class="hide" id="update_recent_tables" href="index.php?ajax_request=1&amp;recent_table=1&amp;token=35e5861f291aa5ad349a28288e444bdf"></a></span>`

**Comment by devenbansod** [comment]
You mean you did not get this error on 4.6.3 but are getting it on 4.6.4 ? 

You are using Cookie authentication method right ? The response and popup text point to something being wrong in cookie Authentication/ cookies, that it is not able to authenticate and is redirecting you to index.php (i.e. preventing you from this action). Currently I am not very sure why this can happen in 4.6.4

Anyways, as you have posted the response, could you also post the request parameters which are sent when you select 'BETWEEN' in the drop-down ?

**Comment by Jay04653** [comment]
Hi,

1) I have this error since 4.4 if I not mistaken. But currently I had new error: **No databases selected.**. All other search options are working fine.
2) Example of connection settings:
```
$cfg['Servers'][$i]['verbose'] = 'localhost';
$cfg['Servers'][$i]['host'] = 'localhost';
$cfg['Servers'][$i]['connect_type'] = 'tcp';
$cfg['Servers'][$i]['auth_type'] = 'cookie';
$cfg['Servers'][$i]['pmadb'] = 'phpmyadmin';
$cfg['Servers'][$i]['controluser'] = 'pma';
$cfg['Servers'][$i]['controlpass'] = '[PASS]';
$cfg['Servers'][$i]['controlhost'] = 'localhost';
$cfg['Servers'][$i]['controlport'] = 3306;
```
3) Request data:
```
token:e86e9801ea7112f672f3fc975a3a60c2
ajax_request:1
db:prod
table:groups
column:
            id        
range_search:1
_nocache:1473164154012828037
```
Response data:
```{"success":false,"error":"<div class=\"error\"><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_error\" \/> No databases selected.<\/div>"}```

**Comment by ibennetch** [comment]
While I can't directly reproduce this, with Chrome I don't get the popup prompt asking me for values that I get with other browsers (Safari and Firefox). Can you test with a different browser to see if your problem persists? My guess is that something isn't going well specifically with Chrome here.

I don't get any kind of error even in the JavaScript console.

**Comment by Jay04653** [comment]
Just testes on latest Google Chrome and Firefox.
Got the same problem.

details from Firefox:
Request:
`token:"33ffefd84246e11b60341d85b550f084"`
`ajax_request:"1"`
`db:"prod"`
`table:"acls"`
`column:"++++++++++++id++++++++"`
`range_search:"1"`
`_nocache:"148179468495571892"`

Response:
`redirect_flag:"1"`
`success:false`
`error:"<a class="hide" id="update_recent_tables" href="index.php?ajax_request=1&amp;recent_table=1&amp;token=513c2fc08d10dfdb8670065578091cb4"></a>"`

**Comment by Jay04653** [comment]
Working now, seems fixed in 4.7.0.
